docs/source/examples/predict_bike_trips.ipynb

```diff
@@ -0,0 +1,414 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Predict Bike Trips\n",
+    "\n",
+    "In this example, we build a machine learning application to predict the number of bike trips from a station in the next biking period. This application is structured into three important steps:\n",
+    "\n",
+    "* Prediction Engineering\n",
+    "* Feature Engineering\n",
+    "* Machine Learning\n",
+    "\n",
+    "In the first step, we generate new labels from the data by using [Compose](https://compose.alteryx.com/). In the second step, we generate features for the labels by using [Featuretools](https://docs.featuretools.com/). In the third step, we search for the best machine learning pipeline by using [EvalML](https://evalml.alteryx.com/). \n",
+    "After working through these steps, you will learn how to build machine learning applications for real-world problems like forecasting demand. Let's get started."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from demo.chicago_bike import load_sample\n",
+    "from matplotlib.pyplot import subplots\n",
+    "import composeml as cp\n",
+    "import featuretools as ft\n",
+    "import evalml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will use data provided by Divvy which is a bicycle sharing system in Chicago. In this dataset, we have a record of each bike trip."
```

Comment:
Might be good to link to the original data source in case people want to investigate further.